File: go/docmost/models/workspace_invitation.go
package models

// WorkspaceInvitation from OpenAPI.
type WorkspaceInvitation struct {
	Id *string `json:"id,omitempty"`
	Email *string `json:"email,omitempty"`
	Role *string `json:"role,omitempty"`
	Token *string `json:"token,omitempty"`
	GroupIds *string `json:"groupIds,omitempty"`
	InvitedById *string `json:"invitedById,omitempty"`
	WorkspaceId *string `json:"workspaceId,omitempty"`
	CreatedAt *string `json:"createdAt,omitempty"`
	UpdatedAt *string `json:"updatedAt,omitempty"`
}

File: c#/src/Docmost.Sdk/Models/WorkspaceInvitation.cs
namespace Docmost.Sdk.Models;

public class WorkspaceInvitation
{
    [System.Text.Json.Serialization.JsonPropertyName("id")]
    public string? Id { get; set; }

    [System.Text.Json.Serialization.JsonPropertyName("email")]
    public string? Email { get; set; }

    [System.Text.Json.Serialization.JsonPropertyName("role")]
    public string? Role { get; set; }

    [System.Text.Json.Serialization.JsonPropertyName("token")]
    public string? Token { get; set; }

    [System.Text.Json.Serialization.JsonPropertyName("groupIds")]
    public string? GroupIds { get; set; }

    [System.Text.Json.Serialization.JsonPropertyName("invitedById")]
    public string? InvitedById { get; set; }

    [System.Text.Json.Serialization.JsonPropertyName("workspaceId")]
    public string? WorkspaceId { get; set; }

    [System.Text.Json.Serialization.JsonPropertyName("createdAt")]
    public string? CreatedAt { get; set; }

    [System.Text.Json.Serialization.JsonPropertyName("updatedAt")]
    public string? UpdatedAt { get; set; }

}
